### PR TITLE
Create the learneth profile

### PR DESCRIPTION
**Pull request by bunsenstraat**
Profile.json for the learneth plugin

### CODE REVIEW

File: plugins/learneth/profile.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+ "name": "LearnEth",
+ "displayName": "LearnEth",
+ "description": "Learn Ethereum with Remix !",
```
Comment (ryestew) [inline plugins/learneth/profile.json:4]: Remix ! -> Remix!  ( you have an extra space)

File: plugins/learneth/profile.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ "documentation": "https://github.com/bunsenstraat/remix-learneth-plugin",
+ "version": "0.1.0-beta.3",
+ "methods": [],
+ "kind": "none",
+ "icon": "https://image.flaticon.com/icons/svg/1570/1570493.svg",
```
Comment (ryestew) [inline plugins/learneth/profile.json:9]: I think that this one is a better icon: https://fontawesome.com/icons/book-reader?style=solid
Changing to this icon involves either finding a cdn with the svg on it or downloading the svg and changing it to a base64 image and putting that in the profile.  
But let me know what you think of the book-reader icon.  I think it fits with the other icons more than the icon from flaticon.

Comment (ryestew) [reply plugins/learneth/profile.json:9]: https://upload.wikimedia.org/wikipedia/commons/d/da/Font_Awesome_5_solid_book-reader.svg

Comment (ryestew) [reply plugins/learneth/profile.json:9]: Don't use the base64 - but put the image in a folder in the remix-workshops repo and then put a link to the file in the profile.

Comment (bunsenstraat) [reply plugins/learneth/profile.json]: I can't link the icon in the repo. If you test it it won't be there unless it's in master. So it put it in IPFS. Hope that works.

File: plugins/learneth/profile.json (hunk 1 above)
Comment (bunsenstraat) [reply plugins/learneth/profile.json]: done

File: plugins/learneth/profile.json (hunk 2 above)
Comment (ryestew) [reply plugins/learneth/profile.json]: We can't load it from IPFS.  Put it in the repo and I'll test it manually ( with the browser tools).

**Pull request merged**